app/controllers/admin/tutorials_controller.rb

```diff
@@ -28,4 +38,32 @@ def destroy
   def tutorial_params
     params.require(:tutorial).permit(:tag_list)
   end
+
+  def playlist_params
+    params.require(:tutorial).permit(:playlist_id)
+  end
+
+  def tutorial_info_params(tutorial)
+    { title: tutorial.title,
+      description: tutorial.description,
+      thumbnail: tutorial.thumbnail,
+      playlist_id: tutorial.id }
+  end
+
+  def video_params(video)
+    { title: video.title,
+      description: video.description,
+      thumbnail: video.thumbnail,
+      video_id: video.id }
+  end
```

Comment:
Maybe these are going to become joined then it alleviates the SRP bump above in favor of creating a combined class.